java/core/lance-jni/src/traits.rs

```diff
@@ -61,10 +61,13 @@ impl FromJString for JString<'_> {
 }
 
 pub trait JMapExt {
+    #[allow(dead_code)]
```

Comment:
@eddyxu there were other methods here marked with `dead_code` so I am guessing you want to keep these?

Reply:
Yes, let's ignore Java Jni for now